css/style.css

```diff
@@ -1,55 +1,58 @@
-html {
-    font-size: 120%;
-    margin: 60px;
-  }
+body {
+  font-size: 120%;
+  margin: 60px;
+  margin: 0;
+  padding: 0;
+  text-align: center;
+  font-family: cursive;
+}
+
+footer {
+  text-align: left;
+  background-color: pink;
+  border: 3px solid purple;
+  border-radius: 5px;
+}
+
+/* fonts */
+
+h1 {
+  color: mediumorchid;
+  font-family: cursive;
+  font-size: 60px;
+}
+
+h2 { 
+  color: mediumorchid;
+  font-family: cursive;
+}
+
+section {
+  max-width: 780px;
+  margin: 0 auto;
+}
+
+/* class selectors */
+
+.main-text {
+  text-align: justified;
+}
```

Comment:
Rather than needing to add the class to each paragraph element you can just add the class to an element that wraps around all of them.